sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -14,43 +14,50 @@
     </item>
     <item>
       <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <Description>Time at which the most recent data was read (TAI unix seconds).</Description>
       <IDL_Type>double</IDL_Type>
       <Units>second</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>ux</EFDB_Name>
-      <Description>x-axis wind speed.</Description>
+      <EFDB_Name>speed</EFDB_Name>
+      <Description>Median wind speed (x, y, z) measured over a period on the order of a minute.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>m/s</Units>
-      <Count>1</Count>
+      <Count>3</Count>
+    </item>
+    <item>
+      <EFDB_Name>speedStdDev</EFDB_Name>
+      <Description>Standard deviation of wind speed (x, y, z) measured over a period on the order of a minute.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s</Units>
+      <Count>3</Count>
     </item>
     <item>
-      <EFDB_Name>uy</EFDB_Name>
-      <Description>y-axis wind speed.</Description>
+      <EFDB_Name>speedMagnitude</EFDB_Name>
+      <Description>Median wind speed magnitude measured over a period on the order of a minute.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>m/s</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>uz</EFDB_Name>
-      <Description>z-axis wind speed.</Description>
+      <EFDB_Name>speedMaxMagnitude</EFDB_Name>
+      <Description>Maximum wind speed magnitude measured over a period on the order of a minute.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>m/s</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>ts</EFDB_Name>
-      <Description>Sonic temperature (air temperature measured sonically). This has poor absolute accuracy (it can be off by several degrees) but good time resolution.</Description>
+      <EFDB_Name>sonicTemperature</EFDB_Name>
+      <Description>Sonic temperature (air temperature measured sonically); median of values measured over a period on the order of a minute. This has poor absolute accuracy (it can be off by several degrees) but good time resolution.</Description>
```

Comment:
Please remove " but has good time resolution" because that doesn't apply anymore, since we're collecting the median of values now.

Reply:
Good point. However, I do want to tell users that the point of sonic temperature is that it has good time resolution. I propose to reword it as follows:

Median sonic temperature (air temperature measured sonically). Sonic temperature has poor absolute accuracy (it can be off by several degrees) but good time resolution.

and also add sonicTemperatureStdDev, to give a better picture of the fluctuations.